strongbox-data-service/src/main/java/com/orientechnologies/orient/object/jpa/OJPAObjectDatabaseTxEntityManager.java

```diff
@@ -0,0 +1,375 @@
+package com.orientechnologies.orient.object.jpa;
+
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.EntityTransaction;
+import javax.persistence.FlushModeType;
+import javax.persistence.LockModeType;
+import javax.persistence.Query;
+import javax.persistence.TypedQuery;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.metamodel.Metamodel;
+
+import com.orientechnologies.orient.core.id.ORecordId;
+import com.orientechnologies.orient.core.metadata.schema.OClass;
+import com.orientechnologies.orient.object.db.OObjectDatabaseTx;
+
+/**
+ * @author Sergey Bespalov
+ *
+ */
+public class OJPAObjectDatabaseTxEntityManager implements EntityManager
```

Comment:
There is OJPAEntityManager inside orientdb-object-2.2.16.jar. Why we implement simply EntityManager interface rather than extending that class and modifying transaction management?

It looks like you just copied code from there and deal like re-involve the bicycle.